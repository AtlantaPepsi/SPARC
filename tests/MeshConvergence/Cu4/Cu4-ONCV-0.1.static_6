***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Cu:
      0.9882651394       0.9500437390       0.9854007162
      0.5311585887       0.9423479712       0.5054895152
      0.9293054969       0.4556304936       0.4885723786
      0.4720456043       0.4781589886       0.0261939929
