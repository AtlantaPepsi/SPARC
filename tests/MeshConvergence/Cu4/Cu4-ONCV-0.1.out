***************************************************************************
*                       SPARC (version Oct 01, 2020)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Oct 18 18:26:34 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.8 6.8 6.8 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 68 68 68
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001
CHEB_DEGREE: 50
NSTATES: 50
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Cu4-ONCV-0.1
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 1
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  1
Mesh spacing                       :  0.1 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Cu4-ONCV-0.1.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  76
Atom type 1  (valence electrons)   :  Cu 19
Pseudopotential                    :  ../../Psp_for_tests/Cu.psp8
Pseudocharge radii of atom type 1  :  6.50 6.50 6.50 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  786.85 MB
Estimated memory per processor     :  786.85 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
